src/install/vm/deployment/previous.mdx

```diff
@@ -0,0 +1,74 @@
+---
+componentType: appInfoConfigOption
+optionType: deployment 
+headingText: Upload previously detected  vulnerabilities through Dependabot Bulk Import 
+---
+
+Import previously detected issues so that New Relic has a complete view of the vulnerabilities detected by Dependabot by performing a bulk import.
+
+## Prerequisites
+
+- A New Relic [license key](/docs/accounts-partnerships/install-new-relic/account-setup/license-key) for the account you want to report data to.
+- A Github Personal Access Token (PAT) with these permissions: `repo AND read:org`
+- The name of your Organization on github.com (for example `newrelic` for github.com/newrelic)
+
+## Bulk import vulnerabilities
+
+Bulk import is initiated with an HTTP POST request. Your GitHub PAT is stored in-memory during import and then discarded. It is never persisted to non-volatile storage.
+
+<Callout variant="caution">
+Running bulk import more than once is not recommended. New Relic will remove duplicate imported data in our vulnerability management UI, but queries to the underlying log data will show duplicate data.
+</Callout>
+
+A bulk import HTTP POST request consists of the following. Replace `<NewRelicIngestKey>`, `<GithubPAT>`, and `<OrganizationName>` with the appropriate values:
+
+### Post:
+
+```
+https://security-ingest-processor.service.newrelic.com/v1/security/import/dependabot
+```
+
+### HEADERS
+
+`Api-Key`: your New Relic Ingest License Key
```

Comment:
the formatting of this needs a little work; it's not line breaking them. 

Also just do 'New Relic license key', and link to it if you need to.